### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ExcplitBugle** [xmrig/xmrig#3046]
Could I embed XMRIG in the browser or is there any embed miner that I could use?

**Comment by Spudz76** [comment]
Not really, all current browsers block anything even close to that sort of activity.

**Comment by snipeTR** [comment]
Technically possible, but if you have enough technical ability to do so, it's a very bad way to make money.
